mmv1/third_party/terraform/tests/resource_access_context_manager_access_policy_test.go.erb

```diff
@@ -164,3 +165,34 @@ resource "google_access_context_manager_access_policy" "test-access" {
 }
 `, org, title)
 }
+
+func testAccAccessContextManagerAccessPolicy_scopedTest(t *testing.T) {
+	org := getTestOrgFromEnv(t)
+	project := getTestProjectFromEnv()
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAccessContextManagerAccessPolicyDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAccessContextManagerAccessPolicy_scoped(org, project, "scoped policy"),
+			},
+			{
+				ResourceName:      "google_access_context_manager_access_policy.test-access",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccAccessContextManagerAccessPolicy_scoped(org, project, title string) string {
+	return fmt.Sprintf(`
+resource "google_access_context_manager_access_policy" "test-access" {
+  parent = "organizations/%s"
+  title  = "%s"
+  scopes = ["projects/%s"]
```

Comment:
Does the use of the scope field mean that there can now be multiple access policies per org?

Reply:
Yes, so long as the scopes are mutually exclusive

Reply:
Why would we want to wait, sorry?